ui/public/locales/zh/translation.json

```diff
@@ -0,0 +1,27 @@
+{
+  "createdSince": "开始",
+  "cronWorkflow": "定时工作流",
+  "entrypoint": "入口",
+  "error": "错误",
+  "failed": "失败",
+  "finishedBefore": "结束",
+  "label": "标签",
+  "labels": "标签",
+  "login": "登录",
+  "logs": "日志",
+  "namespace": "命名空间",
```

Comment:
curious how you chose some of these translations -- did you use the official k8s docs translations as a starting point? Ideally we should match those where possible

I ask because most of these words are technical terms and so they don't translate easily. In other languages I know well enough (e.g. Russian and Spanish), almost none of these translate easily, and many technical words are loan words from English too.

Admittedly I do not have much experience as a localization writer/editor, so I don't really know the norms or best practices therein (I am familiar with adding _support_ for i18n like with React components as you did here, but not in writing translations myself).
I know there are some localization tools that allow users to "rate" translations and suggest alternatives as a feedback mechanism, which can help the iteration loop.

Reply:
Yes, as you mentioned. The Kubernetes dashboard will be a good example which we can learn from. The `namespace` usually be translated to `命名空间`.